legate/core/partition.py

```diff
@@ -430,3 +430,9 @@ def construct(
                 index_space, self, index_partition
             )
         return region.get_child(index_partition)
+
+    def import_partition(self, partition: LegionPartition) -> None:
+        index_partition = partition.index_partition
+        runtime.partition_manager.record_index_partition(
+            index_partition.parent, self, index_partition
```

Comment:
It seems a little odd that this function takes both `index_partition` and `index_partition.parent` as individual parameters. Is that unavoidable?

Reply:
[473a832](https://github.com/nv-legate/legate.core/pull/699/commits/473a832f369229b882464d3e232bd40fa579eec2)